settings.py

```diff
@@ -100,9 +100,10 @@
     'contribute': 'https://github.com/thundernest/thunderbird-website',
     'mozorg.home': 'https://www.mozilla.org/',
     'legal.fraud-report': 'https://www.mozilla.org/about/legal/fraud-report/',
-    'legal.index': 'https://www.mozilla.org/about/legal/',
-    'privacy': 'https://www.mozilla.org/privacy/',
-    'privacy.notices.websites': 'https://www.mozilla.org/privacy/websites/#cookies',
+    'legal.index': 'https://www.mozilla.org/en-US/about/legal/terms/mozilla/',
+    'guidelines': 'https://www.mozilla.org/en-US/about/governance/policies/participation/',
```

Comment:
Technically these are supposed to be alphabetical but I'll mess with it later :P